ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PartyAllocationLedgerEntry.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.stores.ledger
+
+import com.daml.ledger.participant.state.v1.ParticipantId
+import com.digitalasset.ledger.api.domain.PartyDetails
+
+//TODO BH consider removing duplication of this class
+sealed abstract class PartyAllocationLedgerEntry() extends Product with Serializable {
```

Comment:
The entries are missing record time fields.

Reply:
recordTime added